apps/judicial-system/web/src/routes/Judge/CourtRecord/CourtRecord.tsx

```diff
@@ -108,6 +108,10 @@ export const CourtRecord: React.FC = () => {
         attendees += `${wc.judge.name} ${wc.judge.title}\n`
       }
 
+      if (wc.registrar) {
+        attendees += `${wc.registrar.name} ${wc.registrar.title}\n`
+      }
+
```

Comment:
This is all that is needed for the registrar to show up in the pdf. See earlier comment.